node/src/components/block_accumulator/block_acceptor.rs

```diff
@@ -129,7 +130,29 @@ impl BlockAcceptor {
         &mut self,
         finality_signature: FinalitySignature,
         peer: Option<NodeId>,
+        validator_slots: u32,
     ) -> Result<Option<FinalitySignature>, AcceptorError> {
+        /// Returns an error if the peer has sent too many finality signatures.
+        fn check_signatures_from_peer_bound(
```

Comment:
Minor suggestion:
Please consider moving this function out and covering it with a simple UT.

Reply:
Done